rust/scx_rustland_core/assets/bpf.rs

```diff
@@ -70,7 +68,6 @@ pub struct QueuedTask {
     pub pid: i32,              // pid that uniquely identifies a task
     pub cpu: i32,              // CPU where the task is running (-1 = exiting)
     pub sum_exec_runtime: u64, // Total cpu time
-    pub nvcsw: u64,            // Voluntary context switches
```

Comment:
Does this reduction actually make a difference? Depending on the frequency of the classification, reading it from proc can be a lot more expensive.

Reply:
In terms of overhead it's negligible, but the point is that it's a very specific metric required just by a single particular scheduler, potentially we may have other schedulers based on `scx_rustland_core` that don't need this metric at all.

Adding more and more metrics could lead to a bloated `QueuedTask` and it'd like to keep it as small as possible. Especially for the metrics that are already available in user-space. Sure, it's probably slower reading it from procfs, but it's only affecting the task classifier that doesn't need to be extremely fast, so worst case it'll be a bit slower at classifying interactive tasks vs regular tasks.